test/integration/relay/connection.test.js

```diff
@@ -473,7 +481,727 @@ if (helper.sequelize.dialect.name === 'postgres') {
 
         let lastResult = await query(nextResult.data.user.tasks.edges[2].cursor);
         verify(lastResult, lastThree);
-        expect(lastResult.data.user.tasks.pageInfo.hasNextPage).to.equal(false);
+        expect(lastResult.data.user.tasks.pageInfo.hasNextPage).to.equal(true);
```

Comment:
Yeah all of the ones verifying that we've reached the last page of results should now be reverted back to original state.